pkgs/development/python-modules/tubeup/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, internetarchive
+, fetchPypi
+, youtube-dl
+, docopt
+, isPy27
+}:
+
+buildPythonPackage rec {
+  pname = "tubeup";
+  version = "0.0.19";
+
+  disabled = isPy27;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "6e3ebbf677a43018bfd71070919187bd57120010b0d708c0494c0f17bb72e84e";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py --replace "docopt==0.6.2" "docopt"
+  '';
+
+  propagatedBuildInputs = [ internetarchive docopt youtube-dl ];
+
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

Reply:
Thanks for reviewing! I think upstream tests are broken. I'll try to submit a patch later, and update this PR.

Reply:
I opted to add `pythonImportsCheck` for now at least.

I created a PR upstream to fix the tests https://github.com/bibanon/tubeup/pull/127, but unfortunately I don't have much free time nowadays to finish it